File: Guanabara - PHP/exercicios-PHPOO/Aula12-Polimorfismo/Mamifero.php
<?php
include_once 'Animal.php';
class Mamifero extends Animal {
    //Atributo
    protected $corPelo;
    
    //Métodos implementados usando polimorfismo de sobreposição
    public function alimentar() {
        echo '<p>Mamando...</p>';
    }
    public function emitirSom() {
        echo '<p>Som de mamífero</p>';
    }
    public function locomover() {
        echo '<p>Mamífero correndo...</p>';
    }
    
    //Métodos Acessores
    public function getCorPelo() {
        return $this->corPelo;
    }
    public function setCorPelo($corPelo): void {
        $this->corPelo = $corPelo;
    }
}

File: Java/Meus exercícios POO/Aula13.Polimorfismo.Sobrecarga/src/main/java/sobrecarga/Mamifero.java
package sobrecarga;
public class Mamifero extends Animal {
    //Atributo
    protected String corPelo;
    
    //Polimorfismo por Sobreposição
    @Override
    public void emitirSom() {
        System.out.println("Som de mamífero");
    }
    
}
